src/main/webapp/script.js

```diff
@@ -65,11 +65,12 @@ function displayPotentialMatchInfo(pmID) {
     let carouselContainer = document.getElementById("carousel-inner");
     let numPhotos = 5; //hard coded value for now, once blobstore works, this will be the amount of images the user has uploaded
```

Comment:
Use `const` instead of `let` where you can. It looks to me like `numPhotos` won't be updated later, so it can be `const`. There is at least one other place where this is also true.